configure-ios.sh

```diff
@@ -0,0 +1,10 @@
+# sample configure command for ios
```

Comment:
I also don't think this belongs in the repo. A command example like this can be documented on the wiki or the JDK project web page.